datacontract/export/dbt_converter.py

```diff
@@ -72,9 +72,14 @@ def to_dbt_sources_yaml(data_contract_spec: DataContractSpecification, server: s
     adapter_type = None
     if found_server is not None:
         adapter_type = found_server.type
-        source["database"] = found_server.database
-        source["schema"] = found_server.schema_
+        if adapter_type == "bigquery":
+            source["database"] = found_server.project
+            source["schema"] = found_server.dataset
+        else:
+            source["database"] = found_server.database
+            source["schema"] = found_server.schema_
 
+    source["tables"] = []
```

Comment:
The purpose is the same as https://github.com/datacontract/datacontract-cli/pull/541. We want to display information such as which project or dataset the individual tables in the dbt source are in, and who the owner is, at the top of the yaml.